lib/router.js

```diff
@@ -51,6 +51,10 @@ module.exports = (config) => {
     path: paths.translations + '/__lng__/__ns__.json'
   });
 
+  router.use(hof.middleware.deepTranslate({
+    translate: i18n.translate.bind(i18n)
+  }));
+
   router.use(mixins(fields, {
     translate: i18n.translate.bind(i18n)
```

Comment:
This is the same when instantiating wizard too - both give you the option to pass in a custom translate function but fall back to `req.translate`